doc/changelogs/CHANGELOG_V13.md

```diff
@@ -33,6 +34,155 @@
   * [io.js](CHANGELOG_IOJS.md)
   * [Archive](CHANGELOG_ARCHIVE.md)
 
+<a id="13.4.0"></a>
+## 2019-12-17, Version 13.4.0 (Current), @MylesBorins
+
```

Comment:
Missing `This is a security release.` line.

Reply:
Fixed